### DISCUSSION TITLE
How to Manage Assets?

### DISCUSSION THREAD

**Discussion by brunoalano** [nandorojo/solito#46] [Q&A]
What's the ideal way to manage assets (images & fonts) on the monorepo structure? Should I copy file on multiple places? Should I symlink? Or it's possible to create a package called `assets`, and then request files from there?

**Comment by nandorojo** [comment]
i’ve been wondering this for myself tbh. I’ve just been copying it over for the `public` folder. you could also just `import` assets inline though, right? the only things global assets matter for should be `favicon` and splash screens/app icons right? and for the rest you can colocate files with the components that import them

**Reply by brunoalano** [reply]
I see. Probably I will follow with this: keep platform-specific assets on "apps", and then use component-specific assets inside the "packages/app".

Do you have a sample boilerplate using React Navigation + Auth? I saw on some thread that you are interested in creating a premium boilerplate, and I do have interest on paying for it (U$0 ~ U$100/mo).

**Reply by nandorojo** [reply]
there have been a number of requests for an auth example, including #39. If I can get some monthly sponsorships I can make a firebase auth template that uses it react-native-firebase too.

**Reply by brunoalano** [reply]
@nandorojo Done, started sponsoring.

> there have been a number of requests for an auth example

On my case, i don't need a full demo, some a recommended approach for different patterns.
